src/js/views/patients/patient/sidebar/patient-sidebar.scss

```diff
@@ -35,8 +35,15 @@
   }
 }
 
-.patient-sidebar__item + .patient-sidebar__item {
-  margin-top: 4px;
+.patient-sidebar__item {
```

Comment:
I don't think we should necessarily nest these
`.is-empty` is fine to nest since it is a state modifier, but I don't think  `.patient-sidebar__item + .patient-sidebar__item` should be.